modules/integration/pom.xml

```diff
@@ -107,4 +107,28 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <profiles>
+    <profile>
+      <id>webindex-dev-server</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>java</goal>
+                </goals>
+                <phase>compile</phase>
+                <configuration>
```

Comment:
When a I ran the dev server a lot of junk was logged.  Maybe if the classpath scope was set to test here it would pick up the log4j config and suppress the annoying logging.  Could also use maven -q option to make maven not log stuff.

http://www.mojohaus.org/exec-maven-plugin/examples/example-exec-or-java-change-classpath-scope.html